connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceMetrics.java

```diff
@@ -104,12 +108,25 @@ class MirrorSourceMetrics implements AutoCloseable {
         replicationLatencyAvg = new MetricNameTemplate(
                 "replication-latency-ms-avg", SOURCE_CONNECTOR_GROUP,
                 "Average time it takes records to replicate from source to target cluster.", partitionTags);
+        replicationOffsetLag = new MetricNameTemplate(
```

Comment:
I guess we have a disagreement about lag? My understanding of lag is: the real LEO of the source cluster partition minus the LEO that has been written to the target cluster. It seems that your definition of lag is: the lag between the mirror task getting data from consumption and writing it to the target cluster?

Reply:
@hudeqi I hear you now - we do have a disagreement about the lag definition and it was an oversight on my end. I will copy your reply over to the mailing thread and will answer there to keep the discussion in one place